routers/users.py

```
@@ -0,0 +1,143 @@
+
+settings = JwtSettings()
+app = FastAPI()
+
+class UsersBase(BaseModel):
+    nickname :Annotated[Optional[str], Field(min_length=2, max_length=20, description="2자 이상 20자 이하의 닉네임")] = None
+    email : Annotated[EmailStr, Field(description="사용자 이메일")]
+    password : Annotated[str, Field(min_length=8, max_length=50)] # 비밀번호 복잡성 로직은 나중에 추가
+    profile_image_url: Optional[HttpUrl] = None
+
+    @classmethod
+    def as_form(
```

Ruff (0.14.11)

Missing return type annotation for classmethod `as_form`

(ANN206)